pages/about/news/[slug].vue

```diff
@@ -0,0 +1,183 @@
+<!-- eslint-disable no-console -->
+
+<script setup>
+// HELPERS
+import _get from 'lodash/get'
+import format from 'date-fns/format'
+import removeTags from '../utils/removeTags'
+
+// GQL
+import ARTICLE_DETAIL from '../gql/queries/ArticleDetail.gql'
+
+const { $graphql } = useNuxtApp()
+
+const { data } = await useAsyncData('news-story', async () => {
```

Comment:
@pghorpade What's the correct string to use in the useAsyncData call? Is it 'news-story', 'news-detail' or something else?

Reply:
We could use either,I would prefer news-detail